### ISSUE TITLE
The system cannot find the file specified - Microsoft Edge WebView2 Runtime Registry path: Computer\HKEY_CURRENT_USER\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}

### ISSUE THREAD

**Issue by FiveNine** [r0x0r/pywebview#881]
### Specification
- pywebview version: 3.6.3
- operating system: Windows 10
- web renderer: edgechromium

### Description  
  
  Get a blank screen without rendering anything. 
  **Code:** 
```
import webview

window = webview.create_window('Woah dude!', 'https://www.google.com')
webview.start(gui='edgechromium', debug=True)
```
 **Console:**
```
[pywebview] [WinError 2] The system cannot find the file specified - Microsoft Edge WebView2 Runtime Registry path: Computer\HKEY_CURRENT_USER\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}        
[pywebview] Using WinForms / Chromium
```
  I went to Windows Registry Editor, and this given path in the console is wrong. It should be Computer\HKEY_CURRENT_USER\\**SOFTWARE**\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}
  
Any solutions?

**Comment by kevinbevers** [comment]
I think this is linked to the new edge version. it was also present in pywebview 3.6.2

**Comment by Endor-8** [comment]
It looks like the same issue I found recently, caused by the WebView2 Runtime stepping into version 100, which breaks the version check used in pywebview `3.6.2`.

However, pywebview version `3.6.3` is supposed to fix the issue (29273799d5e4010c0964cf21dbb403222b8400a1) and so it did for me.
Are you sure you have updated correctly to version `3.6.3`?

**Comment by Endor-8** [comment]
Yes, @kevinbevers, the message is still present in the log because `pywebview` first checks for the version in `HKCU`, which fails, and then in `HKLM`, which now succeeds (for me) in `3.6.3`.
Maybe this log could be suppressed if the version check succeeds in the end.

Incidentally, @FiveNine, you're right about the key in the log being wrong, but looking at `webview.platforms.winforms.py` you can see it's only wrong in the log. The actual key checked includes `SOFTWARE` in the path.
